crates/project-model/src/tests.rs

```diff
@@ -225,18 +223,6 @@ fn rust_project_cfg_groups() {
     check_crate_graph(crate_graph, expect_file!["../test_data/output/rust_project_cfg_groups.txt"]);
 }
 
-#[test]
```

Comment:
I deleted this test because it's really testing a behavior of the stitched sysroot.